src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -237,75 +277,27 @@ const AppWrapper = (props: { children: ReactNode }) => {
 
     try {
       await Agent.agent.start();
+      setIsOffline(false);
     } catch (e) {
-      // @TODO - foconnor: Should specifically catch the error instead of all, but OK for now.
-      setAgentInitErr(true);
-      // eslint-disable-next-line no-console
-      console.error(e);
-      return;
+      const errorStack = (e as Error).stack as string;
+      // If the error is failed to fetch with signify, we retry until the connection is secured
+      if (/SignifyClient/gi.test(errorStack)) {
+        Agent.agent.bootAndConnect().then(() => {
+          setIsOffline(!Agent.agent.isAgentReady());
+        });
+      } else {
+        throw e;
+      }
     }
     dispatch(setPauseQueueIncomingRequest(true));
-
     await loadDatabase().catch((e) => {
       /* TODO: handle error */
     });
 
-    Agent.agent.connections.onConnectionStateChanged((event) => {
-      return connectionStateChangedHandler(event, dispatch);
-    });
-    Agent.agent.signifyNotifications.onNotificationStateChanged((event) => {
-      return keriaNotificationsChangeHandler(event, dispatch);
-    });
-    Agent.agent.credentials.onAcdcStateChanged((event) => {
-      return acdcChangeHandler(event, dispatch);
-    });
-
-    const oldMessages = (
-      await Promise.all([
-        Agent.agent.credentials.getUnhandledIpexGrantNotifications({
-          isDismissed: false,
-        }),
-        Agent.agent.multiSigs.getUnhandledMultisigIdentifiers({
-          isDismissed: false,
-        }),
-      ])
-    )
-      .flat()
-      .sort(function (messageA, messageB) {
-        return messageA.createdAt.valueOf() - messageB.createdAt.valueOf();
-      });
-    oldMessages.forEach(async (message) => {
-      await keriaNotificationsChangeHandler(message, dispatch);
-    });
-    // Fetch and sync the identifiers, contacts and ACDCs from KERIA to our storage
-    // await Promise.all([
-    //   AriesAgent.agent.identifiers.syncKeriaIdentifiers(),
-    //   AriesAgent.agent.connections.syncKeriaContacts(),
-    //   AriesAgent.agent.credentials.syncACDCs(),
-    // ]);
+    setInitialised(true);
   };
 
-  // @TODO - foconnor: We should allow the app to load and give more accurate feedback - this is a temp solution.
-  // Hence this isn't in i18n.
-  if (agentInitErr) {
-    //if (false) {
-    return (
-      <div className="agent-init-error-msg">
-        <p>
-          There’s an issue connecting to the cloud services we depend on right
-          now (KERIA) - please check your internet connection, or if this
-          problem persists, let us know on Discord!
-        </p>
-        <p>
-          We’re working on an offline mode, as well as improving the deployment
-          setup for this pre-production release. Thank you for your
-          understanding!
-        </p>
-      </div>
-    );
-  }
-
-  return <>{props.children}</>;
+  return initialised ? <>{props.children}</> : <></>;
```

Comment:
Jaime recently got changed this - should just be `return <>{props.children}</>;` I don't think we need the `initialised` variable